src/containers/Tenant/Query/QueryEditor/YqlEditor.tsx

```diff
@@ -103,6 +102,17 @@ export function YqlEditor({
             }
         });
 
+        editor.addAction({
```

Comment:
Did you want to add it to editor's menu? If yes, you should add `contextMenuGroupId`, `contextMenuOrder`.